helm/templates/backend-configmap.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "highlight.fullname" . }}-backend
+  labels:
+    app.kubernetes.io/component: backend
+    app.kubernetes.io/part-of: {{ include "highlight.name" . }}
+    {{- include "highlight.labels" . | nindent 4 }}
+data:
+  CLICKHOUSE_ADDRESS: {{ include "highlight.fullname" . }}-clickhouse:9000
+  CLICKHOUSE_DATABASE: default
+  CLICKHOUSE_PASSWORD: ""
+  CLICKHOUSE_USERNAME: default
+  DOPPLER_CONFIG: docker
+  ENVIRONMENT: dev
+  FRONTEND_URI: {{ .Values.frontend.config.frontendUri | quote }}
+  INFLUXDB_BUCKET: dev-bucket
+  INFLUXDB_ORG: dev-org
+  INFLUXDB_SERVER: {{ include "highlight.fullname" . }}-influxdb:8086
+  INFLUXDB_TOKEN: not-a-secure-token
+  KAFKA_SERVERS: {{ include "highlight.fullname" . }}-kafka:{{ .Values.kafka.config.port }}
+  KAFKA_TOPIC: dev
+  OPENSEARCH_INDEX_PREFIX: docker
+  OPENSEARCH_DOMAIN: http://{{ include "highlight.fullname" . }}-opensearch:9200
+  OPENSEARCH_DOMAIN_READ: http://{{ include "highlight.fullname" . }}-opensearch:9200
+  OPENSEARCH_PASSWORD: admin
+  OPENSEARCH_USERNAME: admin
+  OTLP_ENDPOINT: {{ include "highlight.fullname" . }}-collector:4319
```

Comment:
Might need to add `http://` to the front of the OTLP_ENDPOINT or it might attempt to default to localhost:8413 

Had to dig into `StartOTLP()` to discover this